### PR TITLE
VELO-539

### PR DESCRIPTION
**Pull request by SzymonGesicki**
- increase clickable area

### CODE REVIEW

File: lib/src/drawing/sub_features/shape_drawing/painters/shape_painter.dart [hunk 1]
```diff
@@ -42,20 +44,19 @@ base class ShapePainter extends DrawingPainter<ShapeDrawing> {
...
+        final rect = _prepareRectangle(drawing: drawing, padding: padding);
         return rect.contains(point.toOffset);
 
       case Shape.circle:
-        const margin = 1.0;
-        final rect = _prepareCircle(drawing: drawing);
-        final double distanceToCenter = (point.toOffset - rect.center).distance;
-        final double radius = rect.size.magnitude / 2;
-        return distanceToCenter <= (radius + margin);
+        final rect = _prepareCircle(drawing: drawing, padding: padding);
```
Comment (KlaudiuszBI) [inline lib/src/drawing/sub_features/shape_drawing/painters/shape_painter.dart:55]: do you need to point to the argument name, even though both have the same?

File: lib/src/drawing/sub_features/shape_drawing/painters/shape_painter.dart [hunk 2]
```diff
@@ -74,25 +75,57 @@ base class ShapePainter extends DrawingPainter<ShapeDrawing> {
...
     );
 
-    final double x1 = firstDelta.point.x;
-    final double x2 = secondDelta.point.x;
+    final PointDouble point1 = Point(firstDelta.point.x, firstDelta.point.y);
+    final PointDouble point2 = Point(secondDelta.point.x, secondDelta.point.y);
 
-    final double y1 = firstDelta.point.y;
-    final double y2 = secondDelta.point.y;
+    final PointDouble topVertex = PointDouble((point1.x + point2.x) / 2, point2.y);
```
Comment (KlaudiuszBI) [inline lib/src/drawing/sub_features/shape_drawing/painters/shape_painter.dart:81]: maybe this `(point1.x + point2.x) / 2` formula can be place in some utils package

File: lib/src/drawing/sub_features/shape_drawing/painters/shape_painter.dart [hunk 3]
```diff
@@ -74,25 +75,57 @@ base class ShapePainter extends DrawingPainter<ShapeDrawing> {
...
 
-    final PointDouble topVertex = PointDouble((x1 + x2) / 2, y2);
+    final PointDouble middlePoint =
+        Point((point1.x + point2.x + topVertex.x) / 3, (point1.y + point1.y + topVertex.y) / 3);
```
Comment (KlaudiuszBI) [inline lib/src/drawing/sub_features/shape_drawing/painters/shape_painter.dart:84]: as above

File: lib/src/drawing/sub_features/shape_drawing/painters/shape_painter.dart [hunk 4]
```diff
@@ -74,25 +75,57 @@ base class ShapePainter extends DrawingPainter<ShapeDrawing> {
...
 
+  PointDouble _inflateTrianglePoint({
+    required PointDouble point,
+    required PointDouble middlePoint,
+    required double padding,
+  }) {
+    final double dx = point.x - middlePoint.x;
+    final double dy = point.y - middlePoint.y;
+
+    final double distance = sqrt(dx * dx + dy * dy);
```
Comment (KlaudiuszBI) [inline lib/src/drawing/sub_features/shape_drawing/painters/shape_painter.dart:122]: distance can be a reusable function

File: lib/src/drawing/sub_features/shape_drawing/painters/shape_painter.dart (hunk 1 above)
Comment (SzymonGesicki) [reply lib/src/drawing/sub_features/shape_drawing/painters/shape_painter.dart:55]: yes, if it is named, it is always necessary to specify name

File: lib/src/drawing/sub_features/shape_drawing/painters/shape_painter.dart (hunk 2 above)
Comment (SzymonGesicki) [reply lib/src/drawing/sub_features/shape_drawing/painters/shape_painter.dart:81]: these are quite specific cases, point or offset operations exists, but here we only add x values and divide by different values

**Review by KlaudiuszBI — APPROVED** [review]

**Pull request merged**